### PR TITLE
Add Kubernetes helm charts for serge

### PR DESCRIPTION
**Pull request by mms-gianni**
Here is the PR referring issue https://github.com/serge-chat/serge/issues/491 

We are running it on our cluster. It might be noted somewhere to use really big Kubernetes worker nodes to prevent running out of memory. 

The resource limits are set to CPU: 2000m and Memory: 8Gi which is the minimum we've run it. 

To provide a helm repository you'll need to activate GitHub pages to serve the packages and index yaml. It might coexist with an existing page though since index.html is just an optional redirect. 

I've added also our release script to regenerate the helm chart repository. It will require the helm binary. You can run it in github action but I've never done it yet.

### CODE REVIEW

**Comment by mms-gianni** [comment]
Transfering the repository is still an option if you do not want to host the helm repo with al the tar.gz in your main code repository

**Comment by gaby** [comment]
@mms-gianni Thanks for the PR, I will review it this week
 :-)

**Comment by gaby** [comment]
@mms-gianni Overall looks good, i'm making the changes in the repo to use the official `helm` github-action to auto-deploy this into a github pages :-)

**Comment by gaby** [comment]
@mms-gianni I havent forgotten about this PR, been sick for almost a week. Catching up on stuff now. :-)

File: docs/serge-0.1.0.tgz
Comment (gaby) [inline docs/serge-0.1.0.tgz:1]: This file is not needed if we are publishing to github, right?

File: k8s/serge/values.yaml [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+  #      - chart-example.local
+
+resources:
+  # We usually recommend not to specify default resources and to leave this as a conscious
+  # choice for the user. This also increases chances charts run on environments with little
+  # resources, such as Minikube. If you do want to specify resources, uncomment the following
+  # lines, adjust them as necessary, and remove the curly braces after 'resources:'.
+  limits:
+    cpu: 2000m
+    memory: 8Gi
```
Comment (gaby) [inline k8s/serge/values.yaml:73]: This limit should be like 16GB

File: k8s/serge/values.yaml [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+  requests:
+    cpu: 1000m
+    memory: 4Gi
+
```
Comment (gaby) [inline k8s/serge/values.yaml:77]: Can we add a field to add all gpus? Disabled by default

**Comment by gaby** [comment]
@mms-gianni I'm going to merge this. I will work on adding support for github-pages/deploying the chart in a different PR.

The github-action that I added is linting the chart, and installing the chart in a cluster.

**Comment by gaby** [comment]
Thanks for the awesome PR, really appreciate it

**Pull request merged**